tagstudio/src/qt/ts_qt.py

```diff
@@ -401,6 +401,17 @@ def start(self) -> None:
         select_all_action.setToolTip("Ctrl+A")
         edit_menu.addAction(select_all_action)
 
+        clear_select_action = QAction("Clear Selection", menu_bar)
+        clear_select_action.triggered.connect(self.clear_select_action_callback)
+        clear_select_action.setShortcut(
+            QtCore.QKeyCombination(
+                QtCore.Qt.KeyboardModifier(QtCore.Qt.KeyboardModifier.ControlModifier),
+                QtCore.Qt.Key.Key_D,
+            )
+        )
+        clear_select_action.setToolTip("Ctrl+D")
```

Comment:
It would make more sense to me if the key shortcut was <kbd>Esc</kbd> instead of <kbd>Ctrl</kbd> + <kbd>D</kbd>
```suggestion
        clear_select_action.setShortcut(QtCore.Qt.Key.Key_Escape)
        clear_select_action.setToolTip("Esc")
```

Reply:
<kbd>Esc</kbd> does seem more intuitive. I've changed the shortcut accordingly.